src/clj/rems/db/test_data.clj

```diff
@@ -223,10 +223,10 @@
                                      :infourl (get infourl lang)}]))
         result (with-user actor
                  (catalogue/create-catalogue-item!
-                  {:resid (or resource-id (create-resource! {}))
-                   :form (or form-id (create-form! {}))
+                  {:resid (or resource-id (create-resource! {:organization organization}))
```

Comment:
Should these handle still the default i.e. no organization? Or is the intended direction to always require it?

Reply:
if organization is nil, the defaulting inside create-resource!, create-form! etc. takes over. they all default to `"default"`